### DISCUSSION TITLE
does zoraxy support NTLM protocol ?

### DISCUSSION THREAD

**Discussion by Nonobis** [tobychui/zoraxy#423] [General]
Hello, 
i wanted to known if zoraxy support NTLM protocol ? i have a self hosted Azure Dev Ops i would like to expose by reverse proxy ...

**Comment by tobychui** [comment]
@Nonobis No, we don't support proxying Microsoft's weird protocols.

**Reply by Nonobis** [reply]
Ok .... Thanks for your time ... i known , i known it's a weird protocol ....
